modules/gitops-release-notes-1-10-0.adoc

```diff
@@ -0,0 +1,120 @@
+
+* With this update, the default Argo CD instance in the `openshift-gitops` namespace has restricted permissions for non-admin users by default. This improves security because non-admin users no longer have access to sensitive information. However, as an administrator, you can set permissions and grant non-admin users access to the resources managed by the default `openshift-gitops` Argo CD instance by configuring your Argo CD RBAC. This change only applies to the default {gitops-title} instance. link:https://issues.redhat.com/browse/GITOPS-3032[GITOPS-3032]
+
+* With this update, the default installation namespace for {gitops-title} Operator has been changed to its own namespace called `openshift-gitops-operator`. You can still choose the old default installation namespace, `openshift-operators`, through a drop-down menu available in the OperatorHub UI at installation time. You can also enable cluster monitoring on the new namespace by selecting the check box, which makes the Operator's performance metrics accessible within the {product-title} web console. link:https://issues.redhat.com/browse/GITOPS-3073[GITOPS-3073]
++
+[NOTE]
+====
+The {gitops-title} Operator's metrics are only available when the Operator is installed in the default namespace, `openshift-gitops-operator`.
+====
+
+* Before this update, there was no option for scaling the application controller replicas dynamically. Now, you can dynamically scale the number of application controllers based on the number of clusters managed by each application controller. link:https://issues.redhat.com/browse/GITOPS-3287[GITOPS-3287]
+
+* Before this update, there was no option for choosing an algorithm for distributing the destination clusters equally across the different application controller shards. Now, you can set the sharding algorithm to `round-robin` parameter, which distributes clusters equally across the different application controller shards so that the synchronization load is spread equally among the shards. link:https://issues.redhat.com/browse/GITOPS-3288[GITOPS-3288]
+
+* With this update, the {gitops-title} Operator exports custom metrics that allow you to track the performance of the Operator. The following are the exported metrics:
+** `active_argocd_instances_total` - The number of Argo CD instances currently managed across the cluster.
+** `active_argocd_instances_by_phase{phase="<_PHASE>"}` - The number of managed Argo CD instances in the given phase.
+** `active_argocd_instance_reconciliation_count{namespace="<_YOUR-DEFINED-NAMESPACE>"}` - The number of times the instance in a given namespace is reconciled.
+** `controller_runtime_reconcile_time_seconds_per_instance{namespace="<_YOUR-DEFINED-NAMESPACE>"}` - The distribution of reconciliation cycles by their duration for the instance in a given namespace.
```

Comment:
IBM guidelines:
If a list item has two parts, do not use a hyphen or dash to separate the parts. Use a definition list to define terms, and use a period or colon to separate items in other lists. Be consistent with the punctuation marks that you apply.

So I would suggest changes similar to this:
```suggestion
** `active_argocd_instances_total`: This shows the number of Argo CD instances currently managed across the cluster.
** `active_argocd_instances_by_phase{phase="<_PHASE>"}`: This shows  the number of managed Argo CD instances in the given phase.
** `active_argocd_instance_reconciliation_count{namespace="<_YOUR-DEFINED-NAMESPACE>"}`: This shows the number of times the instance in a given namespace is reconciled.
** `controller_runtime_reconcile_time_seconds_per_instance{namespace="<_YOUR-DEFINED-NAMESPACE>"}`: This metric displays the distribution of reconciliation cycles by their duration for the instance in a given namespace.
```

Reply:
Done!